src/main/java/com/speaker/service/validator/message/MessageFromAccountNamesValidatorImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.speaker.service.validator.message;
+
+import com.speaker.dto.MessageDTO;
+import com.speaker.dto.ValidatorError;
+import com.speaker.entities.EntityField;
+import com.speaker.repository.AccountRepository;
+import com.speaker.service.validator.AbstractValidator;
+import com.speaker.service.validator.Validator;
+import com.speaker.service.validator.type.ErrorType;
+import com.speaker.service.validator.type.FieldType;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import static com.speaker.service.util.StringParser.splitBySpace;
+import static java.util.Objects.isNull;
+
+@Component
+@RequiredArgsConstructor
+public class MessageFromAccountNamesValidatorImpl extends AbstractValidator implements Validator<MessageDTO> {
+    private final AccountRepository accountRepository;
+
+    @Override
+    public ValidatorError validate(EntityField entityField) {
+        if (isNull(entityField.getField().toString())) {
+            return createValidatorError(ErrorType.EMPTY);
+        }
+        if (splitBySpace(entityField.getField().toString()).isEmpty()) {
```

Comment:
extract in variable `entity Field.getField()`

Reply:
Comments accepted, corrections made.